src/stage_node.cpp

```diff
@@ -343,6 +343,22 @@ void StageNode::cb_object_setpose_srv(
   }
 }
 
+void StageNode::cb_get_dyn_objects([[maybe_unused]] const std::shared_ptr<std_srvs::srv::Trigger::Request> request,
+                                   std::shared_ptr<std_srvs::srv::Trigger::Response> response)
+{
+  if(objects_.empty()){
+    response->success = false;
+    response->message = "You first need to define some obstacles!";
+  }else{
+    std::string msg = "Dynamic Objects are: ";
+    for(const auto& object: this->objects_){
+      msg.append(object->name() + " ");
```

Comment:
That is really cumbersome when we can also send arrays of strings via ROS.  Also I think we should do this in a separate PR; this is already good from the size.